root/account/PreferencesSaved.js

```diff
@@ -21,7 +21,7 @@ const PreferencesSaved = ({$c}: Props) => (
   <StatusPage title={l('Preferences')}>
     <p>
       {l('Your preferences have been saved. Click {link|here} to continue to your user page.',
-        {__react: true, link: $c.user ? '/user/profile/' + $c.user.name : '/register'})}
+        {__react: true, link: $c.user ? '/user/' + $c.user.name : '/register'})}
```

Comment:
Technically the issue is still there in that anyone can create a user named `profile/whatever` or `reosarevok/whatever` and have it link to any user they want. The name has to be URI-encoded:
```suggestion
        {__react: true, link: $c.user ? '/user/' + encodeURIComponent($c.user.name) : '/register'})}
```